src/task/render.js

```diff
@@ -0,0 +1,44 @@
+let taskDivContainer = document.getElementsByClassName("taskDiv")[0];
+
+const clearTaskDiv = () => {
+  taskDivContainer.innerText = "";
+};
+
+export const renderTasks = (project, onCheck) => {
+  // Clears my task div
+  clearTaskDiv();
+
+  // For loop for searching the active projects tasks and then adding them to page
+  for (let i in project.tasks) {
+    let task = project.tasks[i];
+    let taskElement = document.createElement("div");
+    let taskTitle = document.createElement("h5");
+    let taskDescription = document.createElement("p");
+    let checkbox = document.createElement("input");
+
+    taskDescription.innerText = task.description;
+    taskTitle.innerText = task.title;
+
+    taskElement.appendChild(taskTitle);
+    taskElement.appendChild(taskDescription);
+    taskElement.appendChild(checkbox);
+    taskElement.setAttribute("class", "taskDetails");
+    taskElement.setAttribute("data-id", i);
+
+    taskTitle.setAttribute("class", "taskTitle");
+
+    taskDescription.setAttribute("class", "taskDescription");
+
+    if (task.checked) {
+      checkbox.setAttribute("checked", task.checked);
+    }
```

Comment:
Little explanation here:
`<input type="checkbox" checked="false" />` will render a checked checkbox. the checked attribute should appear only when the checkbox is checked. So:

`<input type="checkbox" />` => Not checked
`<input type="checkbox" checked="XXX" />` => Checked
`<input type="checkbox" checked />` => Checked

That is why the `if` here  check if the task is checked before adding the attribute.

**Note:** on line 33 we are setting the fact that the checkbox is checked to the value checked in the task. This means that the checkbox does not change right away. Scenario: checkbox is not checked. I check it. The event listener (below, lines 38-40) will be called. The `onCheck` callback will be called. In this method, the state should be mutate so the task checked property is now true. After a new render, the checkbox will be checked.